student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,7 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    const char *hard_coded_username = my_username();
+    char *conf_username = malloc_username_from_conf_file();
```

Comment:
@kbiggs what happens to the string allocated with `malloc_username_from_conf_file`?